src/components/elements/Map-mapbox/components/OverviewMapArea.tsx

```diff
@@ -205,6 +205,7 @@ const OverviewMapArea = ({
         polygonFromMap={polygonFromMap}
         shouldBboxZoom={!shouldRefetchPolygonData}
         modelFilesData={modelFilesData?.data}
+        pdView={true}
```

Comment:
Just some question:
Did you test this with users who are not Monitoring Partners? What should happen with them? Are they able to create versions?

Reply:
If you are not a project monitoring partner, you should not have access to the project or those functions.